asammdf/blocks/v4_blocks.py

```diff
@@ -4493,19 +4493,23 @@ def __init__(self, **kwargs):
 
         if self.comment.startswith("<HDcomment"):
             comment = self.comment
-            comment_xml = ET.fromstring(comment)
-            common_properties = comment_xml.find(".//common_properties")
-            if common_properties is not None:
-                for e in common_properties:
-                    name = e.get("name")
-                    if name == "author":
-                        self.author = e.text
-                    elif name == "department":
-                        self.department = e.text
-                    elif name == "project":
-                        self.project = e.text
-                    elif name == "subject":
-                        self.subject = e.text
+            try:
+                comment_xml = ET.fromstring(comment)
+            except ET.ParseError as e:
+                logger.error("could not parse header block comment; %s", e)
```

Comment:
Please use f-strings or .format method. I hate % interpolation with a passion.

Reply:
makes sense, I will go for f-strings since it only needs to support python3